iina/MPVController.swift

```diff
@@ -664,6 +664,18 @@ class MPVController: NSObject {
     case MPV_EVENT_COMMAND_REPLY:
       let reply = event.pointee.reply_userdata
       if reply == MPVController.UserData.screenshot {
+        let code = event.pointee.error
+        guard 0 <= code else {
+          let error = String.init(cString: mpv_error_string(code))
```

Comment:
```suggestion
          let error = String(cString: mpv_error_string(code))
```

Reply:
I see my inexperience with Swift is showing. Fixed.